### PR TITLE
TimescaleDB connection page

### LINKED ISSUE #2423: [Feature]TimescaleDB support
I would like to request support for [TimescaleDB](http://www.timescale.com/index.html) as netdata backend, which is an extension on top of Postgres. Thanks!

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

Fixes #2423

This PR updates the main `README.md` file to mention TimescaleDB more explicitly. It also adds a new page at `backends/TIMESCALE.md` to explain how the connection works, with links to @mahlonsmith's program, @runejuhl's setup, and the blog post on the TimescaleDB website.

@cakrit — I think this covers what we mentioned on our call a while back. If not, please let me know.

I think I'll work with Jennifer on creating a blog post around this effort as well to celebrate our community a bit more.

##### Component Name

docs

### CODE REVIEW

**Comment by joelhans** [comment]
@cakrit Done.

**Comment by joelhans** [comment]
Hey @vlvkobal & @thiagoftsm, could one of you review this? Just some new content, but it relates to backends. Thank you!

File: README.md [hunk 1]
```diff
@@ -394,7 +394,7 @@ This is what you should expect from Netdata:
...
--   **time-series dbs** - can archive its metrics to **Graphite**, **OpenTSDB**, **Prometheus**, **AWS Kinesis**, **MongoDB**, **JSON document DBs**, in the same or lower resolution (lower: to prevent it from congesting these servers due to the amount of data collected). Netdata also supports **Prometheus remote write API** which allows storing metrics to **Elasticsearch**, **Gnocchi**, **InfluxDB**, **Kafka**, **PostgreSQL/TimescaleDB**, **Splunk**, **VictoriaMetrics** and a lot of other [storage providers](https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage).
+-   **time-series dbs** - can archive its metrics to **Graphite**, **OpenTSDB**, **Prometheus**, **AWS Kinesis**, **MongoDB**, **JSON document DBs**, in the same or lower resolution (lower: to prevent it from congesting these servers due to the amount of data collected). Netdata also supports **Prometheus remote write API** which allows storing metrics to **Elasticsearch**, **Gnocchi**, **InfluxDB**, **Kafka**, **PostgreSQL**, **TimescaleDB**, **Splunk**, **VictoriaMetrics**, and a lot of other [storage providers](https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage).
```
Comment (thiagoftsm) [inline README.md]: Hi @joelhans , 
I verified that you changed `**PostgreSQL/TimescaleDB**` to `**PostgreSQL**, **TimescaleDB**`, this is not officially real, we do not have PostgreSQL exporter yet, the original text is due a reference present in TimescaleDB:

```
Powered by PostgreSQL

Leverage the reliability, maturity, and operational efficiency of one of the world's most popular databases.
```
For more details, please access  https://www.timescale.com/

Comment (thiagoftsm) [reply README.md]: @vlvkobal I was reading TimescaleDB and I verified that is works with SQL, did we already try to use this specific exporter connecting direct to PostgreSQL?

Comment (joelhans) [reply README.md]: Interesting. Well, I can restore that to the way it was. Or, maybe write **PostgreSQL via TimescaleDB**?

Comment (vlvkobal) [reply README.md]: As I understand, TimescaleDB can use the same protocol as PostgreSQL. We don't provide a direct connector. Only through Prometheus Remote Write API.

Comment (joelhans) [reply README.md]: Hi @thiagoftsm — I reverted this to its original form, so it reads **PostgreSQL/TimescaleDB** again.

File: backends/README.md [hunk 2]
```diff
@@ -32,13 +32,15 @@ So, although Netdata collects metrics every second, it can send to the backend s
     -   **prometheus** is described at [prometheus page](prometheus/) since it pulls data from Netdata.
 
     -   **prometheus remote write** (a binary snappy-compressed protocol buffer encoding over HTTP used by
-        **Elasticsearch**, **Gnocchi**, **Graphite**, **InfluxDB**, **Kafka**, **OpenTSDB**, **PostgreSQL/TimescaleDB**,
-        **Splunk**, **VictoriaMetrics**, and a lot of other [storage
+        **Elasticsearch**, **Gnocchi**, **Graphite**, **InfluxDB**, **Kafka**, **OpenTSDB**, **PostgreSQL**, **TimescaleDB**, **Splunk**, **VictoriaMetrics**, and a lot of other [storage
```
Comment (thiagoftsm) [inline backends/README.md]: Hi @joelhans ,

You also split PostgreSQL and TimescaleDB here, I think it is safe to return here.

Comment (joelhans) [reply backends/README.md]: @thiagoftsm Reverted here as well.

**Comment by joelhans** [comment]
@vlvkobal or @cakrit, this should be ready to be merged with one more approval.

**Pull request merged**